docs/how-to-guides/authentication-with-dex-gangway.md

```diff
@@ -250,20 +250,18 @@ Configuring an API server to use the OpenID Connect authentication plugin requir
 
 * Dex is accessible to both your browser and the Kubernetes API server.
 
-To reconfigure the API server with specific flags, edit the `kube-apiserver` DaemonSet as follows:
-
-```bash
-kubectl -n kube-system edit daemonset kube-apiserver
-```
-
-Add the following CLI arguments to the API server pod:
-
-```bash
---oidc-issuer-url=https://dex.YOUR.CLUSTER.DOMAIN.NAME
---oidc-client-id=gangway
---oidc-username-claim=email
---oidc-groups-claim=groups
-```
+To reconfigure the API server with specific flags, add following snippet to your cluster configuration:
```

Comment:
```suggestion
To configure the API server with specific flags, add following snippet to your cluster configuration:
```

Reply:
Assuming you can change that in a created cluster (but won't see why not, except some helm issue... like version number needs to change if you change params to apply them? Hope not :)).

Reply:
You can change it just fine on already created cluster. It will even automatically do a rollback if you specify a bad flag :wink:

Reply:
Oh, and now that I read again... reconfigure still makes sense. Ignore this! :)

Reply:
Thanks!